content/docs/1.10.1/important-notes/_index.md

```diff
@@ -38,6 +39,76 @@ For the full release note, see [here](https://github.com/longhorn/longhorn/relea
     - [V2 Data Engine Replica Rebuild QoS](#v2-data-engine-replica-rebuild-qos)
     - [V2 Data Engine Volume Expansion](#v2-data-engine-volume-expansion)
 
+## Warning
+
+While the upgrade process typically migrates resources automatically to the new `v1beta2` version, there are scenarios, such as backup system interruptions during the upgrade where some Custom Resource (CR) objects may remain in the deprecated `v1beta1` format.
+
+If these CRs are not fully migrated to `v1beta2`, the Longhorn Manager may fail to operate correctly. A common error message indicating this issue looks like:
+
+```
+Upgrade failed: cannot patch "backingimagedatasources.longhorn.io" with kind CustomResourceDefinition: CustomResourceDefinition.apiextensions.k8s.io "backingimagedatasources.longhorn.io" is invalid: status.storedVersions[0]: Invalid value: "v1beta1": missing from spec.versions; v1beta1 was previously a storage version, and must remain in spec.versions until a storage migration ensures no data remains persisted in v1beta1 and removes v1beta1 from status.storedVersions
+```
+
+### Recommended Action
+
+After upgrading Longhorn, verify that all `*.longhorn.io` CustomResourceDefinitions (CRDs) have been fully migrated to `v1beta2`. This ensures system stability and avoids runtime errors.
```

Comment:
Same as above. `After upgrading Longhorn` is confusing. Need to mention from which version to which one.

Reply:
BTW, the description is a bit weird. If the upgrade to v1.10.x succeeds, the following operations are not required.

Reply:
Seems that CRD storedVersion cleanup is required. Added some notes in the warning section.